README.md

```diff
@@ -1,7 +1,14 @@
-## About
+# About
 Current repository contains web interface for ostis platform ([here](https://github.com/ostis-ai) you can find all ostis modules).
 
-## Installation
+# Installation
+## Using Docker
+Running image:
+```sh
+cd sc-web
+docker run --rm -it -p 8000:8000 ostis/sc-web <any additional arguments to app.py> 
```

Comment:
Could you please add info about `--network=host` in case of local sc-server run?